bin/ember-template-lint.js

```diff
@@ -60,7 +60,11 @@ function printErrors(errors, invocationOptions) {
 }
 
 function lintFile(linter, filePath, moduleId) {
-  let source = rw.readFileSync(filePath, { encoding: 'utf8' });
+  let toRead = filePath === STDIN ? process.stdin.fd : filePath;
+
+  // TODO: swap to using get-stdin when we can leverage async/await
+  let source = fs.readFileSync(toRead, { encoding: 'utf8' });
+
```

Comment:
There is a *lot* more going on [in the `rw` package](https://github.com/mbostock/rw/blob/master/lib/rw/read-file-sync.js) are we **absolutely** sure this is correct? My instinct is that DYI pipes could attract dragons that a package like `rw` has already tamed.

Reply:
Ya, I reviewed that code a bit. There are a few specific things that we should avoid that are done in `rw` but the largest one is that if a file that is specified does not exist, we should **not** just assume it means that we want stdin.

Reply:
Yeah, possibly. I want to use [get-stdin](https://github.com/sindresorhus/get-stdin) instead (its a bit simpler to reason about, seems more "popular", etc), and will make an issue to migrate to that (only reason I didn't do it here was because it is promise based, and I'd rather rewrite this to use `async`/`await` than `.then`).